### PR TITLE
Finishing secret number homework

### PR DESCRIPTION
**Pull request by Stephgg**

### CODE REVIEW

File: Stephanie_Gerbracht-Stephgg/secret_number.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		puts "Congrats, you're a total winner!"
+		break		
+	else
+		puts "Nope, you have " + guesses_left.to_s + " left"
+	end
+		
+	if secret_number > guess 
+		puts "P.S that number is too low, guess higher"
+	else 
+		puts "Try a higher number!"
```
Comment (dladowitz) [inline Stephanie_Gerbracht-Stephgg/secret_number.rb]: I think you meant "Try a lower number" since you told them to guess higher on like 63

**Comment by dladowitz** [comment]
Nice Job :+1:

**Comment by Stephgg** [comment]
Aagh.  Once I got the code to work I played around with the wording too
much, I missed that.  Should I go in and change it?

Thanks!!

On Sun, Mar 8, 2015 at 6:54 PM, David Ladowitz <notifications@github.com>
wrote:

> Merged #57 <https://github.com/ga-students/bewd_sf_8_homework/pull/57>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-students/bewd_sf_8_homework/pull/57#event-248264923>
> .
>



-- 
Stephanie Gerbracht
(415) 578-8391

**Comment by dladowitz** [comment]
It's no big deal. I'd focus on the API homework instead.

**Pull request merged**